scrapers/Only3xGirls.yml

```diff
@@ -0,0 +1,32 @@
+name: Only3xGirls
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - only3xgirls.com
+      - purebj.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title:
+        selector: //div[@class="sv_update_title"]/h1
+      Date:
+        selector: //ul[@class="dotsbetween"]//span[contains(.,"since")]/following-sibling::text()
+        postProcess:
+          - parseDate: 01/02/2006
+      Performers:
+        Name: //ul[@class="dotsbetween"]//a
+      Details:
+        selector: //div[@class='set-desc']
+        concat: "\n\n"
```

Comment:
the concat doesnt really do anything since there is a single div

if we wanted to  maintain the format a bit
```yaml
      Details:
        selector: //div[@class='set-desc']/*[local-name()="em" or local-name()="p"]
        concat: "\n\n"
```
looks ok from a few scenes but we would need to check if  only `p` and `em` elements are enough or something else is also used
```

Reply:
Yup that tests out well :-)